toolkits/global/packages/global-forms/README.md

```diff
@@ -140,216 +217,258 @@ You can summarise errors using a top level `errorSummary` property. Each error i
+
+### When to use this form element
+
+Use the text input field when you need to let users enter text that’s no longer than a single line, such as their name or email address.
+
+### How it works
+
+All text inputs must have visible labels. Labels are positioned above the input and are written in sentence case.
+
+#### Avoid placeholder text
+
+The text input component does not support placeholder text. 
+
+Placeholder text:
+
+- disappears when a user enters information - which makes it hard for some people to remember what it said
+- can be mistaken by some users for information they’ve entered
+
+Instead, add a hint above the input, when you need one. 
+
+#### Hints
+
+![Hints](https://user-images.githubusercontent.com/15365576/210446292-047bb015-34b4-4b50-a29a-567d95b41faa.png)
+
+The `hint` property adds hint text under the main label text but *inside* the `<label>`. This means it is automatically available to screen reader software.
+
+```
+"hint": "Add the funder(s) of your research for this manuscript"
+```
+
+Always place hint text above the text input instead of outside it. This will prevent users from mistaking it as a value and skipping the field.
+
+#### Data properties
 
-`Select` fields (using the `<select>` element) define the user’s options with an `options` property, which must be an array. The `selected` property is a Boolean:
+In addition to these top-level properties, you can add data properties as a `dataAttrs` array, which can be useful for unit testing.
 
 ```json
-"options": [
-    {
-        "label": "Horse",
-        "value": "Horse"
-    },
-    {
-        "label": "Platypus",
-        "value": "Platypus",
-        "selected": true
+{
+    "template": "globalFormText",
+    "label": "Your email address",
+    "id": "your-email",
+    "name": "your-email",
+    "dataAttrs": {
+        "test": "someValue"
     }
-]
+}
 ```
 
+This would create the following attribution: `data-test: someValue.`
+
+#### Datalist
+
+Text input fields can have a `datalist` property, which lets you implement autocomplete. This takes 2 properties: `id` and `options` (an array):
+
+```json
+"datalist": {
+    "id": "my-datalist",
+    "options": [
+        "monkey",
+        "horse",
+        "emu",
+        "platypus",
+        "cockroach",
+        "whale"
+    ]
+}
+```
+
+This builds a standard `datalist` field, with `options`, and associates it with the input. If a `datalist` already exists in the markup, provide just the `id` and forego the `option` property. If you want to use a custom-built autocomplete solution using JavaScript, omit the `datalist` property from the field altogether.
+
+
+## Radios
+
+![Radios](https://user-images.githubusercontent.com/15365576/210527456-7584ec3c-2c80-43f5-abd9-6583e6896bdd.png)
+
+### When to use this form element
+
+Use the radio input when users need to pick a single option from a list.
```

Comment:
Technically, it would always be more than one `<input>`...
```suggestion
Use the radios field (`globalFormRadios`) when users need to pick a single option from a list.
```